Libplanet/Tx/TxId.cs

```diff
@@ -87,9 +86,21 @@ public ImmutableArray<byte> ByteArray
             }
         }
 
-        public static bool operator ==(TxId left, TxId right) => Operator.Weave(left, right);
+        public static bool operator ==(TxId left, TxId right) => left.Equals(right);
 
-        public static bool operator !=(TxId left, TxId right) => Operator.Weave(left, right);
+        public static bool operator !=(TxId left, TxId right) => !left.Equals(right);
+
+        public bool Equals(TxId other)
+        {
+            return ByteArray.SequenceEqual(other.ByteArray);
+        }
+
+        public override bool Equals(object obj)
+        {
+            return obj is TxId other && Equals(other);
+        }
```

Comment:
Trivial stylistic suggestions:

```suggestion
        public bool Equals(TxId other) =>
            ByteArray.SequenceEqual(other.ByteArray);

        public override bool Equals(object obj) =>
            obj is TxId other && Equals(other);
```